VARIABLES.md

```diff
@@ -598,15 +622,15 @@ Default:  "{{ schema_registry.properties }}"
 
 ### kafka_rest_user
 
-Only use to customize Linux User Rest Proxy Service runs with. User must exist on host.
+Only use to customize Linux User Rest Proxy Service runs with. Default user is cp-kafka-rest
 
 Default:  "{{kafka_rest_default_user}}"
 
 ***
 
 ### kafka_rest_group
 
-Only use to customize Linux Group Rest Proxy Service user belongs to. Group must exist on host.
+Only use to customize Linux Group Rest Proxy Service user belongs to. Default group is confluent
```

Comment:
Modify this variable to customize the Linux Group that the Rest Proxy Service user belongs to. Default group is confluent.